coffee_vending_machine/beans.py

```diff
@@ -0,0 +1,49 @@
+
+
+class Bean:
+    def __init__(self, max_quantity) -> None:
+        self.current_quantity = 0
+        self.max_quantity = max_quantity
+
+    def add_beans(self, quantity):
+        if quantity + self.current_quantity > self.max_quantity:
+            raise Exception("Total quantity is exceeding max")
+        self.current_quantity += quantity
+
+    def remove_beans(self, quantity):
+        self.current_quantity = - quantity
+        if self.current_quantity < 0:
+            quantity = 0
```

Comment:
Part of interfaces, because you are defining behaviour. This should not be part of Bean class.

Read this -> https://www.calhoun.io/how-do-interfaces-work-in-go/